corehq/apps/locations/static/locations/ko/location_drilldown.async.js

```diff
@@ -107,6 +107,11 @@ function LocationModel(data, root, depth, func, withAllOption) {
   this.func = typeof func !== 'undefined' ? func : LocationModel;
   this.withAllOption = typeof withAllOption !== 'undefined' ? withAllOption : true;
 
+  this.enable = function() {
```

Comment:
I don't have strong preferences on the name, I just pulled it from the knockout data bind.  I believe there it means "enable this element".
```
data-bind="enable: enable()
```
Looking at this again, I think semantic naming makes more sense - something like `children_are_editable`.  I'm also not sure why it uses `every`, seems like it should be `any`, but that's outside the scope of this PR to change.